### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by Hrashi100ms**
Removed token endpoint
Removed -v1 app implementation
Added Krisp doc
Added Prebuilt component details

### CODE REVIEW

File: docs/get-started/v2/get-started/features/noise-cancellation.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: Noise Cancellation
+nav: 3.9
+---
+
+The Noise cancellation feature is an invaluable tool designed to enhance the audio quality in scenarios such as conferences, live streams, and recordings where unwanted background noise can degrade the listening experience. By leveraging advanced Artificial Intelligence (AI) algorithms, this feature intelligently identifies and suppresses extraneous sounds, ensuring clear and crisp audio output.
+
+## Key Benefits
```
Comment (mntrspace) [inline docs/get-started/v2/get-started/features/noise-cancellation.mdx:8]: This is fluff piece which is usually not necessary in technical documentations. Point out any key benefits or features, if any.

File: docs/get-started/v2/get-started/features/noise-cancellation.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+title: Noise Cancellation
+nav: 3.9
+---
+
+The Noise cancellation feature is an invaluable tool designed to enhance the audio quality in scenarios such as conferences, live streams, and recordings where unwanted background noise can degrade the listening experience. By leveraging advanced Artificial Intelligence (AI) algorithms, this feature intelligently identifies and suppresses extraneous sounds, ensuring clear and crisp audio output.
```
Comment (mntrspace) [inline docs/get-started/v2/get-started/features/noise-cancellation.mdx:6]: either both N and C are capital or both are in small.

Comment (mntrspace) [reply docs/get-started/v2/get-started/features/noise-cancellation.mdx:6]: Keep them in capital

Comment (mntrspace) [reply docs/get-started/v2/get-started/features/noise-cancellation.mdx:6]: You can extend this to the entire doc

File: docs/get-started/v2/get-started/features/noise-cancellation.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+<Note type='warning'>
+**IMPORTANT** <br/>
+This is an add-on paid feature, for details check [100ms pricing page](https://www.100ms.live/pricing/) 
+</Note>
+
+### Enabling the Noise cancellation
+
+Noise cancellation can be configured from the 100ms dashboard.
+
+#### Enabling the Noise cancellation
```
Comment (mntrspace) [inline docs/get-started/v2/get-started/features/noise-cancellation.mdx:42]: Add a note below this section that by default, noise cancellation will be enabled for everyone in the session. If they don't want that, the next section will be helpful

File: docs/get-started/v2/get-started/features/noise-cancellation.mdx (hunk 2 above)
Comment (Hrashi100ms) [reply docs/get-started/v2/get-started/features/noise-cancellation.mdx:6]: Got it. Will make N and C caps only.

File: docs/get-started/v2/get-started/features/noise-cancellation.mdx (hunk 1 above)
Comment (Hrashi100ms) [reply docs/get-started/v2/get-started/features/noise-cancellation.mdx:8]: Yes, Felt so will remove second line and just keep first one. 
Key benefits are added below this line

File: docs/get-started/v2/get-started/features/noise-cancellation.mdx (hunk 3 above)
Comment (Hrashi100ms) [reply docs/get-started/v2/get-started/features/noise-cancellation.mdx:42]: got it. adding a note

File: docs/get-started/v2/get-started/features/noise-cancellation.mdx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    <source src="/docs/docs/v2/noise-cancellation/experience-nc.mp4" type="video/mp4" />
+</video> 
+
+</StepsContainer>
+
+### Frequently Asked Questions (FAQ)
+
+1. **Is Noise cancellation a chargeable feature?**
+    
+    Yes, 100ms Noise cancellation is charged based on per peer usage minutes. This means that for every peer that is enabling noise cancellation during session, their individual usage minutes will be aggregated. For more check [100ms pricing page](https://www.100ms.live/pricing/) or reach out to us.
```
Comment (mntrspace) [inline docs/get-started/v2/get-started/features/noise-cancellation.mdx:86]: For more information, please check [100ms pricing page](https://www.100ms.live/pricing/).

Comment (Hrashi100ms) [reply docs/get-started/v2/get-started/features/noise-cancellation.mdx:86]: Ack

**Comment by Hrashi100ms** [comment]
making changes in new pr

**Pull request closed without merging**